README.md

```diff
@@ -230,28 +230,28 @@ it would be misleading to explain one without the other.
 We cover material in logical order, so that if you do read the earlier sections first,
 later sections on the interactions of tax and compensation will be clearer.
 
-The first sections are about how companies work and how they grow:
+We start with [**Equity Compensation Basics**](#equity-compensation-basics): What compensation
+and equity compensation are.
```

Comment:
"What compensation and equity are, and why equity is used by some companies as compensation."

Reply:
Fixed